components/net/http_loader.rs

```diff
@@ -1917,11 +1921,11 @@ async fn http_network_fetch(
     // TODO when https://bugzilla.mozilla.org/show_bug.cgi?id=1030660
     // is resolved, this step will become uneccesary
     // TODO this step
-    if let Some(encoding) = response.headers.typed_get::<ContentEncoding>() {
-        if encoding.contains("gzip") {
-        } else if encoding.contains("compress") {
-        }
-    };
+    // if let Some(encoding) = response.headers.typed_get::<ContentEncoding>() {
+    //     if encoding.contains("gzip") {
+    //     } else if encoding.contains("compress") {
+    //     }
+    // };
```

Comment:
It looks like the bug is resolved so I would just remove all of Step 12 and this code as well.